pytest/tests/loadtest/locust/common/social.py

```diff
@@ -11,30 +11,24 @@
 
 from account import TGAS, NEAR_BASE
 import key
-from common.base import Account, CreateSubAccount, Deploy, NearNodeProxy, Transaction
+from common.base import Account, Deploy, NearNodeProxy, Transaction, FunctionCall
 from locust import events, runners
 from transaction import create_function_call_action
 
 
-class SocialDbSet(Transaction):
+class SocialDbSet(FunctionCall):
 
     def __init__(self, contract_id: str, sender: Account):
-        super().__init__()
+        super().__init__(sender, contract_id, "set")
         self.contract_id = contract_id
         self.sender = sender
 
     @abstractmethod
-    def build_args(self) -> str:
+    def build_args(self) -> dict:
```

Comment:
From what I can tell, we're not using `build_args` anywhere else, maybe it's worth removing it altogether from the interface and instead implementing `args` directly in the child classes?

Reply:
well spotted, removing the function makes more sense to me, too